esp/src/src/react/theme.tsx

```diff
@@ -0,0 +1,28 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+export const theme = createMuiTheme({
+    palette: {
+      primary: {
+          main: '#1a9bd7',
+          light: '#66ccff',
+          dark: '#006da5',
+          contrastText: '#f5f5f5'
+      },
+      secondary: {
+          main: '#455a64',
+          light: '#718792',
+          dark: '#1c313a',
+          contrastText: '#fff'
+      },
+    },
```

Comment:
We could declare a "complete", "running", "failed" palette here - which I would then reuse in the stepper?

Reply:
What I have added here is the palette for our overall theme throughout ECLWatch and a secondary for some contrast.
You can customize any of these default theme objects as listed [here](https://material-ui.com/customization/default-theme/)

If you wanted to set a global style for MuiStepIcon you can provide it in the override section. https://material-ui.com/api/step-icon/


This will hinder us unless we always want our stepper to be primary.main round circle with the default check mark otherwise we would use the useStyles hook inside of wuStatus.tsx